.circleci/config.yml

```diff
@@ -35,60 +35,12 @@ jobs:
               docker push valhalla/valhalla:run-latest
             elif [[ ! -z "${CIRCLE_TAG}" ]]; then
               echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+              docker tag valhalla/valhalla:build-latest valhalla/valhalla:build-${CIRCLE_TAG}
               docker push valhalla/valhalla:build-${CIRCLE_TAG}
+              docker tag valhalla/valhalla:run-latest valhalla/valhalla:run-${CIRCLE_TAG}
```

Comment:
if you look above in the build step of this job you'll see regardless of master or tag we always tag the local image with `-latest`. if this was a github tag though that wouldnt have worked because here we have docker push with the git tag suffix on the image tag name.